2015-16/provincial/are/pipeline-spec.yaml

```diff
@@ -0,0 +1,191 @@
+are-2015-16:
+  title: Annual Report Expenditure 2015-16
+  description: Annual Report Expenditure, with Audited Outcome and Final Appropriation.
+  pipeline:
+    - run: load
+      parameters:
+        from: "https://s3-eu-west-1.amazonaws.com/manual-uploads.vulekamali.gov.za/are/PRV+AR+2015-16.xlsx"
+        name: 'are-2015-16'
+        format: 'xlsx'
+      cache: true
+
+    - run: delete_fields
+      parameters:
+        resources: are-2015-16
+        fields:
+        - BudYear
+        - ValueIndi
+        - Combo
+        - GFS1
+        - GFS2
+        - GFS3
+
+    - run: filter
+      parameters:
+        resources: are-2015-16
+        out:
+          - FY_Descript: Variance
+    - run: filter
+      parameters:
+        resources: are-2015-16
+        out:
+          - FY_Descript: Revised payments
+
+    - run: add_computed_field
+      parameters:
+        resources: are-2015-16
+        fields:
+        - operation: format
+          target: Budget Phase
+          with: '{FY_Descript}'
+
+    - run: find_replace
+      parameters:
+        resources: are-2015-16
+        fields:
+        - name: FinYear
+          patterns:
+          - find: (\d{4})_\d{2}
+            replace: \1
+        - name: Budget Phase
+          patterns:
+          - find: ^(Actual Payments)
+            replace: "Audit Outcome"
+          - find: ^Virement and shifting of funds
+            replace: "Final Appropriation"
+        - name: FY_Descript
+          patterns:
+          - find: ^(Actual Payments)
+            replace: Total
+          - find: ^(Adjusted Appropriation)
+            replace: Total
+          - find: ^(Final Appropriation)
+            replace: Total
+        - name: Department
```

Comment:
mmmm not sure why this was in previous examples - please remove from this one

We shouldn't be rewriting department names directly here, only using the department_names processor

Reply:
So, do you think that programme_names processor is enough?

Reply:
See above - we shouldn't be doing replacements on the Department column in the pipeline. Just remove line 64-71

Reply:
Okay, I understand. You can find the changed versions of two files. When whole process is done, I'm going to send you the results as well.

https://github.com/vulekamali/treasury-pipelines/pull/25/commits/9eca39925701ff85e34c1e81d98675d8914119a4